test/e2e/generator/generate.go

```diff
@@ -5,6 +5,7 @@ import (
 	"math/rand"
 	"sort"
 	"strings"
+	"time"
 
 	e2e "github.com/tendermint/tendermint/test/e2e/pkg"
 )
```

Comment:
`testnetCombinations` defined in line 15 was adding the possibility to test "builtin" and "out of process".
This also modified `nodeABCIProtocols` in line 29 (BTW, looks like we are not testing GRPC as it is now).
Do we want to backport this change in structure of the variables controlling the test combinations?

Reply:
gRPC seems to have been purposefully disabled due to https://github.com/tendermint/tendermint/issues/5439. It's hard to judge whether this is still an issue

Reply:
We can look to port more of the e2e functionality later. I kind of want to pick off important chunks first

Reply:
Agreed. Anyway remember that this effort is not in the critical path (i.e., it's not blocking the creation of branch v0.37.x). IOW, it can proceed along with the release QA process